### DISCUSSION TITLE
Last SRB Miner miss some algos

### DISCUSSION THREAD

**Discussion by BadWolf-63** [RainbowMiner/RainbowMiner#2921] [Show and tell]
SRBminer 2.6.9: "Reduced miner binary size: removed algorithms -> 'pyrinhashv2', 'pyrinhash', 'karlsenhash', 'blake3_alephium', 'sha512_256d_radiant', 'sha3d', 'ubqhash', 'progpow_veriblock', 'pufferfish2bmb', 'randomkeva', 'randomgrft', 'randomnevo'"

I have moved the code to SrbMinerMulti244 to keep algos and double mining:
[CRYPTO.zip](https://github.com/user-attachments/files/18241786/CRYPTO.zip)

**Comment by RainbowMiner** [comment]
Ok, good move! I will follow your steps.
